MACD.py

```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as plt
+import datetime
+import yfinance as yf
+
+ticker = 'AAPL'
+
+start_time = datetime.datetime(2020, 1, 1)
+end_time = datetime.datetime(2020, 10, 10)
+
+query = yf.Ticker(ticker)
+
+df = query.history(start=start_time,end=end_time)
+df = df.reset_index()
+
+df = df[['Date', 'Close']]
```

Comment:
Essentially here, you're resetting the index of the DataFrame to series from 0 onwards. Then resetting the DataFrame 'df' back to just 'date' and 'close'. Is it necessary to do so, as line 15 essentially reverts line 13? Why delete all other columns?